services/madoc-ts/src/frontend/shared/layout/LayoutContainer.tsx

```diff
@@ -23,7 +29,9 @@ export const NavIconContainer = styled.div<{ $active?: boolean; $disabled?: bool
   border-radius: 3px;
   padding: 0.5em;
   margin: 0.25em;
-  width: 2.5em;
+  color: #333;
+  width: auto;
+  display: flex;
```

Comment:
Does this affect the existing vertical tab width? (which was 2.5em)

Reply:
Because its just a icon and the height is set, the width looks okay